src/clr-angular/forms/datepicker/providers/date-io.service.ts

```diff
@@ -21,9 +21,15 @@ import {
 import { getNumberOfDaysInTheMonth, parseToFourDigitYear } from '../utils/date-utils';
 
 import { LocaleHelperService } from './locale-helper.service';
+import { ClrDateRange } from '../interfaces/date-range.interface';
+import { DayModel } from '../model/day.model';
 
 @Injectable()
 export class DateIOService {
+  public disabledDates: ClrDateRange = {
```

Comment:
Can we put a comment on this about why these bottom/top values are set by default, and that this could be modified to become an array of ranges, so in the future we could allow for more configuration to disable multiple ranges (like a booking calendar).